### DISCUSSION TITLE
[jupyter-book] How to set jb to use conf.py instead of _conf.yml ?

### DISCUSSION THREAD

**Discussion by selobu** [executablebooks/meta#888] [Q&A]
Hi, I want to customize the conf.py file with some latext code, so I need to give --conf command a python file instead of _conf.yml file

for example

`jupyter-book build . --builder pdflatex --config conf.py`

**Comment by chrisjsewell** [comment]
Heya, I would suggest you simply use `sphinx-build` (https://www.sphinx-doc.org/en/master/man/sphinx-build.html) for this.
Really the only thing that `jupyter-book build` does, is read the `_conf.yml` then pass it through `sphinx-build`
